Added reference kernel fallback for depthwise_conv

lamarrr opened the pull request:
I added the reference kernel fallback for the depthwise_conv EvalQuantizedPerChannel routine

lamarrr committed:
Added reference kernel fallback for depthwise_conv

I added the reference kernel fallback for the depthwise_conv EvalQuantizedPerChannel routine

freddan80 commented on tensorflow/lite/experimental/micro/kernels/cmsis-nn/depthwise_conv.cc:
```diff
@@ -206,7 +207,15 @@ TfLiteStatus EvalQuantizedPerChannel(TfLiteContext* context, TfLiteNode* node,
   }
 #else
-#error ARM_MATH_DSP and ARM_MATH_LOOPUNROLL must be set
+
```
Actually, it would be good for the user to be notified that there's no optimization for this target. You can add a directive for that. For example:
#pragma message("CMSIS-NN optimization for depthwise_conv not available for this target. Using ref kernel.")

lamarrr replied:
done! Please do review

lamarrr committed:
added CMSIS-NN optimization error message

added directive to notify the user of unavailability of CMSIS-NN optimization for the compilation target

lamarrr committed:
Update depthwise_conv.cc

rthadur commented:
gentle ping @petewarden for review ?
